frontend/src/lib/components/apps/editor/SettingsPanel.svelte

```diff
@@ -54,6 +52,22 @@
 
 		return undefined
 	}
+
+	$: {
```

Comment:
@rubenfiszel  I've added this missing migration, some bg scripts of the e-com app were missing recomputeOnInputChanged